diff -Naur tsung-1.3.1/Makefile.in tsung-1.3.1.tpg/Makefile.in
--- tsung-1.3.1/Makefile.in	2009-08-07 08:58:49.000000000 +0000
+++ tsung-1.3.1.tpg/Makefile.in	2010-02-18 20:21:15.000000000 +0000
@@ -20,11 +20,11 @@
 raw_erlang_prefix=@libdir@/erlang/
 
 prefix=$(DESTDIR)@prefix@
-exec_prefix=@exec_prefix@
-bindir=@bindir@
-libdir=@libdir@
-datadir=@datadir@
-datarootdir=@datarootdir@
+exec_prefix=$(DESTDIR)@exec_prefix@
+bindir=$(DESTDIR)@bindir@
+libdir=$(DESTDIR)@libdir@
+datadir=$(DESTDIR)@datadir@
+datarootdir=$(DESTDIR)@datarootdir@
 TEMPLATES_SUBDIR=@TEMPLATES_SUBDIR@
 
 CONFIGURE_DEPENDENCIES=@CONFIGURE_DEPENDENCIES@
